src/global/variables.css

```diff
@@ -0,0 +1,7 @@
+:root {
+  --color-white: #ffffff;
+  --color-gray-lighter: #d6d7d9;
+  --color-gray-lightest: #f1f1f1;
+  --color-gold-light: #f9c642;
+  --font-source-sans: Source Sans Pro, Helvetica Neue, Helvetica, Roboto, Arial, sans-serif;
+}
```

Comment:
This file contains CSS variables that can be used across Components. This is how we can style certain shadow DOM things at a global level.